src/foam/nanos/crunch/ui/CapabilityWizardlet.js

```diff
@@ -42,8 +42,8 @@ foam.CLASS({
         if ( ! this.of ) return null;
 
         var ret = this.of.getAxiomByName('capability') ?
-          this.of.create({ capability: this.capability }, this) :
-          this.of.create({}, this);
+          this.of.create({ capability: this.capability }, this.__subContext__) :
```

Comment:
Please bring this up and resolve with @kgrgreer

Reply:
I think this should just be 'this'.

Reply:
Yeah so before @KernelDeimos and I were having difficulties with it  just being this,  but just tested it  right now with merge to foam master and it is no longer a problem.  Reverting back to just this.